src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <ul class="block stars-1">
+      <li class="stars_star"></li>
```

Comment:
Check requirements in readme
```suggestion
    <ul class="stars stars--1">
      <li class="stars_star"></li>
```

Reply:
fixed it